boards/microbit_v2/Makefile

```
@@ -15,25 +17,57 @@ TOCKLOADER=tockloader
 .PHONY: install
 install: flash
 
+
 .PHONY: flash-debug
 flash-debug: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/debug/$(PLATFORM).elf
-	$(OPENOCD) $(OPENOCD_OPTIONS) -c "program $<; verify_image $<;  reset; shutdown;"
+	$(CARGO_FLASH) --chip nRF52833_xxAA --verify --path $<
+
+.PHONY: flash-debug-probe-rs
+flash/kernel/probe-rs/debug: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/debug/$(PLATFORM).elf
```

```suggestion
.PHONY: flash-debug-probe-rs
flash-kernel-probe-rs-debug: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/debug/$(PLATFORM).elf
```

I don't know exactly what is right but if these match I can be more confident in approving.